priv/perf/apps/load_test/lib/child_chain/transaction.ex

```diff
@@ -30,9 +52,16 @@ defmodule LoadTest.ChildChain.Transaction do
       |> Enum.map(&Map.get(&1, :priv))
       |> Enum.map(&Encoding.to_hex/1)
 
-    tx
-    |> Transaction.sign(keys: keys)
-    |> try_submit_tx(retries)
+    {:ok, blknum, txindex} =
+      tx
+      |> Transaction.sign(keys: keys)
+      |> try_submit_tx(retries)
+
+    outputs
```

Comment:
n00bie comment, but would it be more consistent for public api to return `{:ok, data}` instead?